macosx/CMakeLists.txt

```diff
@@ -414,6 +415,10 @@ set_target_properties(
         OUTPUT_NAME ${MAC_BUNDLE_NAME}
         MACOSX_BUNDLE_GUI_IDENTIFIER "org.m0k.transmission"
         MACOSX_BUNDLE_INFO_PLIST ${CMAKE_CURRENT_SOURCE_DIR}/Info.plist.in
+        XCODE_ATTRIBUTE_CLANG_ENABLE_MODULES YES
+        XCODE_ATTRIBUTE_OTHER_SWIFT_FLAGS ""
+        XCODE_ATTRIBUTE_SWIFT_OBJC_BRIDGING_HEADER "macosx/Transmission-Bridging-Header.h"
```

Comment:
This is Xcode-specific setup. I'm not sure what it translates to but I very much expect it to work for non-Xcode builds too (and not only with Ninja generator).

Reply:
That was the hardest issue.
I feel I've been through the whole documentation of cmake, ninja and swiftc.
I've been struggling for more than two months at making Swift compiling with cmake. :)
Well, too much time on it, I got the resolution for those lines in cfc49a7649cca047a6077ab3a35921ed4ef18089

Reply:
That was the hardest issue.
I feel I've been through the whole documentation of cmake, ninja and swiftc.
I've been struggling for more than two months at making Swift compiling with cmake. :)
Well, too much time on it, I got the resolution for those lines in cfc49a7649cca047a6077ab3a35921ed4ef18089

Reply:
@mikedld @Coeur is this topic resolved?

Reply:
There are essentially three ways to build with cmake:
- `cmake -G "Unix Makefiles"` -- doesn't support Swift (and there are no plans for this)
- `cmake -G Ninja` -- supports Swift
- `cmake -G Xcode` -- supports Swift
Current CmakeLists is compatible with `ninja` and `xcode`. I documented the compatibility under:
># Ninja|Xcode compatibility